components/raftstore/Cargo.toml

```diff
@@ -38,6 +38,7 @@ crc32fast = "1.2"
 crossbeam = "0.8"
 derivative = "2"
 encryption = { path = "../encryption", default-features = false }
+engine_rocks = { path = "../engine_rocks", default-features = false } 
```

Comment:
Should be optional and enabled only when engine rocks is chosen.

Reply:
how? I don't see any current reference of engine_rocks is optional.